modules/get-started/pages/whats-new.adoc

```diff
@@ -2,140 +2,28 @@
 
 * xref:redpanda-cloud:get-started:whats-new-cloud.adoc[]
 * xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]
 
-== Leader pinning
+== Redpanda Console v3.0.0 (beta)
 
-For a Redpanda cluster deployed across multiple availability zones (AZs), xref:develop:produce-data/leader-pinning.adoc[leader pinning] ensures that a topic's partition leaders are geographically closer to clients. Leader pinning can lower networking costs and help guarantee lower latency by routing produce and consume requests to brokers located in certain AZs.
+The Redpanda Console v3.0.0 beta release includes the following updates:
 
-== Iceberg topics
+=== New features
 
-The xref:manage:topic-iceberg-integration.adoc[Iceberg integration for Redpanda] allows you to store topic data in the cloud in the Iceberg open table format. This makes your streaming data immediately available in downstream analytical systems without setting up and maintaining additional ETL pipelines. You can also integrate your data directly into commonly-used big data processing frameworks, standardizing and simplifying the consumption of streams as tables in a wide variety of data analytics pipelines.
+Unified authentication and authorization system. Redpanda Console now supports user impersonation so you can authenticate to Redpanda using the same credentials you use to authenticate to Redpanda Console. This feature simplifies user management and allows you to use the same credentials for both Redpanda and Redpanda Console.
 
-== Mountable topics
+See xref:console:config/security/authentication.adoc[] for more information.
 
-For topics with Tiered Storage enabled, you can unmount a topic to safely detach it from a cluster and keep the topic data in the cluster's object storage bucket or container. You can mount the detached topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster. See xref:manage:mountable-topics.adoc[Mountable topics] for details.
+=== Breaking changes
 
-== Intra-broker partition balancing
+- No more support for the plain login provider. Use a Redpanda SASL/SCRAM user instead.
+- No more support for group-based authorization with OIDC.
+- The `login` stanza in the configuration file has been renamed to `authentication`.
+- The `login.jwtSecret` field in the configuration file has been renamed to `authentication.jwtSigningSecret`.
+- The `schemaRegistry` stanza has been removed from the `kafka` stanza in the configuration file. Use the top-level `schemaRegistry` stanza instead.
+- Role bindings are not longer configured in a separate file. Use the `authorization.roleBindings` stanza instead.
```

Comment:
The schema for these also changed. It used to be something like this:

```yaml
  roleBindings:
    - roleName: admin
      metadata:
        name: Redpanda POC
      subjects:
        - kind: user
          provider: Plain
          name: "e2euser"
```

and now this:

```yaml
authorization:
  roleBindings:
    - roleName: viewer
      users:
        - loginType: basic
          name: weecos
        - loginType: basic
          name: admins
        - loginType: OIDC
          name: "104753799096002778779" # Refers to the sub claim
    - roleName: admin
      users:
        - loginType: basic
          name: admin
```